content/adr/adopt-semantic-versioning.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Adopt semantic versioning for releases"
+date: "2022-04-13"
+decision: "We will adopt semantic versioning for naming our releases"
+status: "accepted"
```

Comment:
Are you sure you want to do this? In my experience, we introduce breaking changes at all times in a project development cycle. Of course, this is unclear because most software we build won't have a public API.

If you want to enhance the definition of what constitutes a "breaking change" for a project being built, please add that in the document. From my reading, I couldn't find anything that clarifies this.

```suggestion
status: "proposed"
```

Also, we can consider merging this as is and open it up for broader comments. For that purpose, please change the status.